src/lib.rs

```diff
@@ -1,9 +1,17 @@
 pub mod frame;
 pub mod invaders;
+pub mod menu;
 pub mod player;
 pub mod render;
 pub mod score;
 pub mod shot;
 
 pub const NUM_ROWS: usize = 20;
 pub const NUM_COLS: usize = 40;
+
+#[derive(PartialEq, Copy, Clone)]
+pub enum Scenes {
```

Comment:
(We're removing this, but...)
* if we were to keep this, it should be moved to a submodule.
* if we were to keep this, the name of an `enum` should correspond to what _a variant is_. For example, if you said "what is a `Game`?" You would answer "it is a `Scene`."

```suggestion
pub enum Scene {
```